src/wanderer/yellowray.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  appearanceRates,
+  canAdventure,
+  getLocationMonsters,
+  itemDropsArray,
+  Location,
+  toMonster,
+} from "kolmafia";
+import { sum } from "libram";
+import { freeFightFamiliarData } from "../familiar/freeFightFamiliar";
+import { garboValue } from "../session";
+import { canWander, DraggableFight, maxBy, UnlockableZones, WandererTarget } from "./lib";
+
+function averageYrValue(location: Location) {
+  const badAttributes = ["LUCKY", "ULTRARARE", "BOSS"];
+  const rates = appearanceRates(location);
+  const monsters = Object.keys(getLocationMonsters(location))
+    .map((m) => toMonster(m))
+    .filter((m) => !badAttributes.some((s) => m.attributes.includes(s) && rates[m.name] > 0));
+
+  if (monsters.length === 0) {
+    return 0;
+  } else {
+    return (
+      sum(monsters, (m) => {
+        const items = itemDropsArray(m).filter((drop) => ["", "n"].includes(drop.type));
+        return sum(items, (drop) => garboValue(drop.drop));
```

Comment:
```suggestion
        return sum(items, (drop) => garboValue(drop.drop, true));
```

to make it use historical values